shared-data/default-theme/html/logs/events/index.html

```diff
@@ -28,20 +28,21 @@
        href="{{ U('/logs/events/?ui_expand=', event.event_id, '#', event.event_id) }}">
       {{ event.message }}
     </a>
+    {%- if event.data and event.data.undo %}
+    <a href="#"
+       title="{{ event.data.undo }}"
+       class="auto-modal eventlog-undo"
+       data-event_id="{{ event.event_id }}"
+       style="font-style: italic;
+              text-decoration: underline;">{{ _('Undo') }}</a>
```

Comment:
Good job implementing text formatting to  expose Undo functionality to user viewing the logs!